packages/odyssey-react-mui/src/Tag.tsx

```diff
@@ -41,22 +192,27 @@ const Tag = ({
   onRemove,
   testId,
   translate,
+  colorVariant = "default",
 }: TagProps) => {
+  const odysseyDesignTokens = useOdysseyDesignTokens();
   const { chipElementType } = useContext(TagListContext);
 
   const renderTag = useCallback(
     (muiProps: MuiPropsContextType) => (
-      <MuiChip
+      <StyledTag
         {...muiProps}
+        as={chipElementType}
```

Comment:
Moving `chipElementType` to the `as` prop doesn't cause any issues? The component still receives the context correctly?

Reply:
since I am using a styled component, there's no component prop, which caused the "Tags as a list" to fail accessibility tests since it was always rendering as a `div`, `as` seems to be working now correctly to pass the correct element context.